tests/elm/ElmLintBearTest.py

```diff
@@ -0,0 +1,56 @@
+from bears.elm.ElmLintBear import ElmLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file_no_unexpected_comma = """
+module UnexpectedComma (..) where
+var =
+"Hello World!"
+"""
+
+
+bad_file_unexpected_comma = """
+module UnexpectedComma (..) where
+va,r =
+"Hello World!"
+"""
+
+
+good_file_no_unexpected_end_of_input = """
+module UnexpectedEndOfInput (..) where
+var =
+100
+"""
+
+
+bad_file_unexpected_end_of_input = """
+module UnexpectedEndOfInput (..) where
+var =
+"""
+
+
+good_file_not_empty = """
```

Comment:
This isnt a particularly good example of a non-empty file, as it has a leading blank line, and a line feed in the middle of an assignment (implicit continuation).  Simplify it, so that it only tests one thing.  Ideally also add a few from https://github.com/coala/coala-bears/issues/1495